docs/html/development/getting-started.rst

```diff
@@ -50,14 +50,14 @@ To run tests locally, run:
 
 .. code-block:: console
 
-    $ tox -e py36
+    $ tox -e py36 -- -n auto
 
-Generally, it can take a long time to run pip's test suite. To run tests in parallel,
-which is faster, run:
+The version of tox given below also works in a similar way, although a bit slower than the version above
```

Comment:
I would say it’s more than a bit slower, or *a lot slower*.

Reply:
> I would say it’s more than a bit slower, or _a lot slower_.

Yeah you're right. Let me make these changes.